src/components/Article.jsx

```diff
@@ -33,23 +35,29 @@ const {articleId} = useParams();
         }) 
     }, [articleId])
 
+useEffect(() => {
+
+})    
+
 if (error) {
     return <p className="warning-text">{errMsg}</p>
 } else if (loading) {
     return <p>Please wait while page loads</p>
 } 
 
 return (
+    <>
     <div className="article-full">
         <h2>{result.title}</h2>
         <p>Penned by {result.author}</p>
         <p>Topic of {result.topic}</p>
-        <p>{dateFormatter(result.created_at)}</p>
+        <p>Published on {dateFormatter(result.created_at)}</p>
         <h3>Article Text:</h3>
```

Comment:
I'm not sure that you need this h3 - it'll be intuitive without

Reply:
agree, it's gone